x-pack/plugins/rollup/public/crud_app/sections/job_create/steps/step_date_histogram.js

```diff
@@ -246,12 +246,11 @@ export class StepDateHistogramUi extends Component {
                 <p>
                   <FormattedMessage
                     id="xpack.rollupJobs.create.stepDateHistogram.sectionDataSourceDescription"
-                    defaultMessage={`
-                      Which field contains the date histogram data and how large should its time buckets be?
+                    defaultMessage=// eslint-disable-line react/jsx-equals-spacing
```

Comment:
Is it really necessary to place message on the next line and to disable eslint rule? It also works without new line:
```suggestion
                    defaultMessage="Which field contains the date histogram data and how large should its time buckets be?
                      Note that smaller, more granular intervals take up proportionally more space.
                      You can also customize the time zone stored with the rolled-up documents.
                      The default time zone is UTC."
```